system/diagnostic_graph_aggregator/README.md

```diff
@@ -3,20 +3,25 @@
 ## Overview
 
 The diagnostic graph aggregator node subscribes to diagnostic array and publishes aggregated diagnostic graph.
-As shown in the diagram below, this node introduces extra diagnostic status for intermediate functional unit.
-Diagnostic status dependencies will be directed acyclic graph (DAG).
+As shown in the diagram below, this node introduces extra diagnostic status for intermediate functional units.
 
 ![overview](./doc/overview.drawio.svg)
 
-## Diagnostics graph message
+## Diagnostic graph structures
 
-The diagnostics graph that this node outputs is a combination of diagnostic status and connections between them.
-This graph consists of an array of diagnostic nodes, and each node has a status and links.
-This link contains an index indicating the position of the node in the graph.
-Therefore, the graph can be reconstructed from the array of nodes using links.
-The following is an example of a message representing the graph in the overview section.
+The diagnostic graph is actually a set of fault tree analysis (FTA) for for each operation mode of Autoware.
```

Comment:
A `for` is unnecessary.

Reply:
Thank you. Fixed it https://github.com/autowarefoundation/autoware.universe/pull/6715/commits/cdaa322c9470d7fce62b64896326b4f028c401be.